js/libs/keycloak-admin-client/src/defs/authenticatorConfigInfoRepresentation.ts

```diff
@@ -16,4 +16,5 @@ export interface ConfigPropertyRepresentation {
   defaultValue?: any;
   options?: string[];
   secret?: boolean;
+  required?: boolean;
```

Comment:
Let's keep the naming consistent with what PatternFly expects.
```suggestion
  isRequired?: boolean;
```
```

Reply:
@jonkoops Thanks for the review.

Do you mean to change the attribute for the whole DTO on the server side? It'd break the naming conventions provided there. 

From my understanding, this should be in sync with the `ConfigPropertyRepresentation` DTO received from the server, right? Otherwise, the mapping between them would not be done correctly. 

Or can you elaborate on your suggestion?

Reply:
Ah, I did not notice this was a change to the representation, my bad. What I meant is that we should pass it around as `isRequired={required}` to React components, as there is already the convention for PatternFly to use the name `isRequired`.

The representation here should remain unchanged, as we don't want to make changes to the API.